### PR TITLE
replace /members with /users

### PR DESCRIPTION
**Pull request by kotesh-arya**
### Design Doc:
https://docs.google.com/document/d/1xvllP-qyZBAHFzp5Oh-7d-bTh_JHX7Ps_KsOTUSkPYE/edit?usp=sharing

### Changes done:
The `/members` route is being used in 3 different pages to use all the Users Data and it is replaces with `/users` route with a similar response:
1. Task creation page
image
2. Online-members page
image

### Test Coverage Stats:
image

### CODE REVIEW

File: task/script.js [hunk 1]
```diff
@@ -435,19 +435,19 @@ async function fetchLevel() {
...
 }
 
-async function fetchMembers() {
+async function fetchUsers() {
   try {
-    const response = await fetch(`${API_BASE_URL}/members`);
+    const response = await fetch(`${API_BASE_URL}/users`);
     const data = await response.json();
-    membersData = data.members;
+    membersData = data.users;
```
Comment (vinit717) [inline task/script.js:442]: This is RDS member data or user data

Comment (kotesh-arya) [reply task/script.js:442]: RDS user data only

**Review by Achintya-Chatterjee — COMMENTED** [review]
tests are failing , kindly look into it

**Comment by kotesh-arya** [comment]
> tests are failing , kindly look into it

Fixed them, thanks for pointing out 👍🏻

**Review by vinit717 — APPROVED** [review]

**Review by Achintya-Chatterjee — APPROVED** [review]

**Pull request merged**